system-int-test/src/cmr/system_int_test/utils/bootstrap_util.clj

```diff
@@ -16,11 +16,26 @@
     [cmr.system-int-test.utils.url-helper :as url]
     [cmr.transmit.config :as transmit-config]))
 
+(defn bulk-index-after-date-time-without-token
```

Comment:
We shouldn't duplicate each method to use/not use the token. Instead we should use multiple arrities to pass in or not pass in a token.

i.e.
```
([date-time]
  (bulk-index-after-date-time date-time {transmit-config/token-header (transmit-config/echo-system-token)})
([date-time headers]
```
Then you can just pass in `nil` or `{}` when you don't want it to use the system token.

Reply:
Yeah, I know. I was expecting this comment in the review. I was just trying to fix it without impacting anything in the original tests to see if it builds successfully...